content/docs/how-to-contribute.md

```diff
@@ -11,56 +11,56 @@ redirect_from:
 
 React is one of Facebook's first open source projects that is both under very active development and is also being used to ship code to everybody on [facebook.com](https://www.facebook.com). We're still working out the kinks to make contributing to this project as easy and transparent as possible, but we're not quite there yet. Hopefully this document makes the process for contributing clear and answers some questions that you may have.
 
-### [Code of Conduct](https://code.facebook.com/codeofconduct)
+### [Code of Conduct](https://code.facebook.com/codeofconduct) {#code-of-conducthttpscodefacebookcomcodeofconduct}
```

Comment:
Found an inconsistent behavior when a heading is also a link.

The correct slug is `codeofconduct`: https://reactjs.org/docs/how-to-contribute.html

Reply:
It's `code-of-conduct`: https://reactjs.org/docs/how-to-contribute.html#code-of-conduct

Reply:
Thanks for catching that though, should be fixed :)